qml/pages/ProgramDelegate.qml

```diff
@@ -26,15 +27,18 @@ ListItem {
     }
 
     Item {
-        height: parent.height - 2 * Theme.paddingSmall
+        id: listContainer
+        //height: parent.height - 2 * Theme.paddingSmall
+        height: title.height + episode.height + time.height + 2*Theme.paddingSmall
         width: parent.width - 2 * Theme.paddingMedium
         anchors.centerIn: parent
 
         Rectangle {
             id: img
             color: "black"
-            height: parent.height
+            height: parent.height - 2*Theme.paddingSmall
```

Comment:
`2 * Theme.paddingSmall`

Reply:
Fixed by resolving the previous comment.